src/xenia/gpu/command_processor.cc

```diff
@@ -1160,10 +1160,15 @@ bool CommandProcessor::ExecutePacketType3_EVENT_WRITE_ZPD(RingBuffer* reader,
             register_file_->values[XE_GPU_REG_RB_SAMPLE_COUNT_ADDR].u32);
     // 0xFFFFFEED is written to this two locations by D3D only on D3DISSUE_END
     // and used to detect a finished query.
-    bool isEnd = pSampleCounts->ZPass_A == xe::byte_swap(0xFFFFFEED) &&
-                 pSampleCounts->ZPass_B == xe::byte_swap(0xFFFFFEED);
+    bool isEndViaZpass = pSampleCounts->ZPass_A == xe::byte_swap(0xFFFFFEED) &&
```

Comment:
`xe::byte_swap(0xFFFFFEED)` can be cached somewhere above since it's used in multiple places already.